client/my-sites/plugins/plugin-details-CTA/style.scss

```diff
@@ -159,3 +199,7 @@
 		@extend %placeholder;
 	}
 }
+
+.ellipsis-menu__menu.popover {
```

Comment:
`z-index` is best declared in `client/assets/stylesheets/shared/functions/_z-index.scss`

Reply:
Implemented at https://github.com/Automattic/wp-calypso/pull/66282/commits/baea81a75bd82b58401be5a46abba55555105248